atlasdb-jepsen-tests/src/jepsen/atlasdb/timestamp.clj

```diff
@@ -109,5 +71,5 @@
                                      (gen/sleep 85)
                                      {:type :info, :f :stop}])))
                     (gen/time-limit 360))
-    :db (create-server)
+    :db (timelock/create-db)
```

Comment:
I'm not 100% sure it would even be a flake (as opposed to a legitimate bug) if you get a violation in the startup stage. There won't be a liveness violation (nemeses not active yet), the history _should_ still be linearizable, and duplicates is always a bad thing.